dbc2val/mapping.yml

```diff
@@ -411,6 +412,7 @@ VCLEFT_rearDoorState:
         fullmapping:
           DOOR_STATE_CLOSED: "false"
           DOOR_STATE_OPEN_OR_AJAR: "true"
+          DOOR_STATE_RELEASING_LATCH: "true"
```

Comment:
@SebastianSchildt - we got warnings on that this value was not handled. I added a line to also consider that value. As this is an example mapping I do not know if exact mapping is that important. As I understand this is an intermediate state so we could theoretically handle it in 3 ways:

- Consider that the door is opened when this is reported
- Consider that it is still closed
- Ignore this state, but then we should rather have a syntax so that we can state that explicitly so we do not get a warning.
Any comments?

Reply:
For the example mapping such details are not important, so the solution in this mapping is fine.  A clean solution might be to have  variant of the mapping transform.  Currently there is "fullmapping", which, as you have noticed will print a warnming if something can not be append, and we have partialmapping, that will just pass through the raw value, if no mapping is found (i.e. this can be used if you only want to remap a subset of states), what we are missing is a mapping transform  that maps, what it can, and silently ignores non-mapped values. Future work :)